src/utils/S3.js

```diff
@@ -0,0 +1,102 @@
+import {
+  S3Client,
+  PutObjectCommand,
+  GetObjectCommand,
+  DeleteObjectCommand,
+} from "@aws-sdk/client-s3";
+import { getSignedUrl } from "@aws-sdk/s3-request-presigner";
+import * as dotenv from "dotenv";
+import crypto from "crypto";
+import path from "path";
+import fs from "fs";
+import mime from "mime-types";
+import logger from "../config/logger/index.js";
+
+dotenv.config();
+
```

Comment:
You can't use `process.env.KEY` without this? As I see, we already did it in  `app.js`. So maybe we can skip this.

Reply:
Okay